### DISCUSSION TITLE
After using "dua i", how to move files or folders that do not need to be deleted?

### DISCUSSION THREAD

**Discussion by arstum** [Byron/dua-cli#182] [Q&A] [answered]
After using "dua i", how to move files or folders that do not need to be deleted?

**Comment by gosuwachu** [comment, marked as answer]
Only the files you mark for deletion will be deleted. Press `?` while running `dua i` to see list of available commands.

**Comment by i2** [comment]
Would it be possible to implement the cut/move files or folders in dua? I'd like that.

**Reply by Alistair1231** [reply]
I agree. I love the simple interface for deleting large files/folders, but sometimes, I use an `.old` folder, or the like, as an intermediary before I delete stuff. For these kinds of situations it would be great to be able to use the same interface as for deleting the rest, which I delete directly.

How to pick the target folder for pasting would have to be considered. Perhaps one could activate the paste mode while on the right pane and then select the target in the left?  
I imagine another hotkey like Ctrl+M while on the right pane, this swaps the user back to the left pane, where you can navigate and select the target with spacebar, like you would normally flag for deletion.

**Reply by Byron** [reply]
Doing this would also mean that the right-hand side is processed so that top-level entries can be moved freely, i.e. none of the entries are contained within each other. I don't think this is currently truly the case and it definitely relies on the deletion to just ignore double-deletion.
For completeness I may add that `Ctrl + T` moves all files into the trash, as a specialised form of 'move'.

**Reply by i2** [reply]
When dealing with important files, I personally don't dare to trash them. I know they'll still exist in trash and can be moved somewhere else from there, but not ideal.
